localisation/00_zuwc_united_localisation.csv

```diff
@@ -2920,16 +2920,17 @@ EVTOPTA_WCPRM_600;As water wish.;;Wie Wasser w
 EVTOPTB_WCPRM_600;No! You have completely lost your hearing, old man!;;Nein! Ihr habt vollkommen das Geh�r verloren, Alter!;;;;;;;;;;;x
 EVTDESC_WCPRM_605;The oracle frowns and cries: "I'm the voice of the water, young one. The rivers want to warn you. If you decide to remain deaf and go against the tide, well, it is not in my power to convince you.";;Das Orakel runzelt die Stirn und schreit: �Ich bin die Stimme des Wassers, J�ngling. Die Fl�sse wollen dich warnen. Wenn du dich entscheidest, taub zu bleiben und gegen den Strom zu schwimmen, nun, dann liegt es nicht in meiner Macht, dich zu �berzeugen.�;;;;;;;;;;;x
 EVTOPTA_WCPRM_605;Old fool!;;Alter Narr!;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+form_the_gorian_empire;Form the Gorian Empire;;;;;;;;;;;;;x
+form_the_gorian_empire_desc;Once the Gorian Empire was the greatest state on Draenor, time to bring its legacy and glory in the brave new world.;;;;;;;;;;;;;x
+has_realm_province_with_5_or_more_settlements_tooltip;Has realm province with 5 or more settlements;;;;;;;;;;;;;x
+create_gorian_empire_tt;�Y[Root.GetBestName]�! creates the �Y[e_gorian_empire.GetName]�!;;;;;;;;;;;;;x
+EVTTITLE_WCOE_100;Reborn in Purple;;;;;;;;;;;;;x
+EVTDESC_WCOE_100;"If we can't return to Draenor, we will rebuild our civilization in full glory in our new world!", with these words [target_imperator.GetTitledFirstName] started [target_imperator.GetHerHis] speech to the young senate and people of [target_new_goria.GetName].\n\nPurple flags were hung all over the capital earlier that day before this unique event. Surrounded by purple and gold to emphasize the former greatness of the old empire, and the current. The Senate declared the beginning of a new era upon Azeroth, the Gorian Era.\n\nThe masses assembled in the square in front of the Senate while they exclaimed loudly, saluting their first new [target_imperator.GetTitle] since their departure from Draenor.;;;;;;;;;;;;;x
```

Comment:
Changed the second paragraph a little
```suggestion
EVTDESC_WCOE_100;"If we can't return to Draenor, we will rebuild our civilization in full glory in our new world!", with these words [target_imperator.GetTitledFirstName] started [target_imperator.GetHerHis] speech to the young senate and people of [target_new_goria.GetName].\n\nPurple flags were hung all over the capital earlier that day before this unique event. The central square was surrounded by purple and gold banners to emphasize the glory of the past and future empire. The Senate declared the beginning of a new era upon Azeroth, the Gorian Era.\n\nThe masses assembled in the square in front of the Senate while they exclaimed loudly, saluting their first new [target_imperator.GetTitle] since their departure from Draenor.;;;;;;;;;;;;;x
```

Reply:
done